patches/misc/IncrementalProperties.sysprop.cpp

```diff
@@ -0,0 +1,201 @@
+// Generated by the sysprop generator. DO NOT EDIT!
+
+#include <IncrementalProperties.sysprop.h>
```

Comment:
files under misc aren't .path files, maybe move them somewhere else or convert them to .patch files if that works